EDU-2191-Component-EN-Network-Layer-Protection (#345)
aziontech/docs: src/content/docs/en/pages/products/edge-firewall/network-layer-protection/network-layer-protection.mdx

When activating the **Network Layer Protection** module in the *Main Settings* of an **Edge Firewall Rule Set**, the conditions (*criteria*) and commands (*behavior*) will be enabled in the **Rules Engine** tab of the Edge Firewall Rule Set.

> :::note

Reviewer: ```suggestion
:::tip
```

In this example, requests originating from countries that are on the blocklist will be blocked unless the `User-Agent` header contains the string "Googlebot".

> :::tip

Reviewer: ```suggestion
:::note
```

The conditions and behaviors available in the Edge Firewall Rules Engine tab depend on the activated modules. Learn more about [Edge Firewall modules](/en/documentation/products/edge-firewall/).
:::

## Network Lists

Through **Network Lists**, you can create, search, or update all Network Lists used in the **Edge Firewall Rules Engine**. Also, you can maintain your own lists, via [Real-Time Manager (RTM)](https://manager.azion.com), through the Network Lists configuration page, or via [Azion API](https://api.azion.com/), through the Network Lists endpoints. 

A single Network List can be associated with more than one Edge Firewall Rule or Rule Set. Whenever a Network List is updated, it'll automatically propagate to all the rules in the associated Network List.

> :::note
> Learn more about [Network Lists](/en/documentation/products/edge-firewall/network-layer-protection/network-lists/).
> :::

Reviewer: ```suggestion
import Button from '~/components/Button.astro';
<Button href="/en/documentation/products/edge-firewall/network-layer-protection/network-lists/" text="go to Network Lists reference" variant="secondary"> 
</Button>
```
